plugins/WPML.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * ACF Pro Plugin.
+ *
+ * @package Junaidbhura\Composer\WPProPlugins\Plugins
+ */
```

Comment:
Change file block comment to reflect WPML plugin.

```suggestion
/**
 * WPML Plugin.
 *
 * @package Junaidbhura\Composer\WPProPlugins\Plugins
 */
```